helpdesk/teams/views.py

```diff
@@ -172,3 +173,49 @@ def get_entries(self) -> QuerySet[Any]:
 
     def get_context_data(self, **kwargs: Any) -> dict[str, Any]:
         return super().get_context_data(entries=self.get_entries(), **kwargs)
+
+
+class TeamAttendanceView(LoginRequiredMixin, SingleTableMixin, ListView):
+    model = Team
+    table_class = TeamAttendanceTable
+
+    def get_queryset(self) -> QuerySet[Any]:
+        return Team.objects.all().prefetch_related(
+            Prefetch(
+                "team_attendance_events",
+                TeamAttendanceEvent.objects.order_by("-created_at")[:1],
+                to_attr="latest_event",
+            )
+        )
+
+
+class TeamAttendanceFormView(LoginRequiredMixin, CreateView):
+    http_method_names = ["get", "post"]
+    model = TeamAttendanceEvent
+    form_class = TeamAttendanceLogForm
+
+    def get_initial(self) -> dict[str, Any]:
+        return {
+            "team": get_object_or_none(Team, tla=self.kwargs["tla"]),
```

Comment:
Issue: Whilst I like the idea, it's probably better to keep the team only in the URL, rather than in the form.